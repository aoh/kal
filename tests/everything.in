# flags: -t 0 -e

12.1.1970 
 - this is the last day

1.1.1970
 - first post!

every Thursday on even weeks: it's even Thursday, shown up to 12.1.
