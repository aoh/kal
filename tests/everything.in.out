Computing last date
Looking at #[comment # flags: -t 0 -e] vs max #[date 4 1 1970 7 1]
Looking at #[event #[date 12 1 1970 1 3] this is the last day] vs max #[date 4 1 1970 7 1]
Looking at #[event #[date 1 1 1970 4 1] first post!] vs max #[date 12 1 1970 1 3]
Looking at #[recurring #[and #[daily 4] #[week-parity even]] it's even Thursday, shown up to 12.1.] vs max #[date 12 1 1970 1 3]
1.1.1970, Thursday, week 1
 - first post!

8.1.1970, Thursday, week 2
 - it's even Thursday, shown up to 12.1.

12.1.1970, Monday, week 3
 - this is the last day
